sample_site/_includes/navigation.njk

```diff
@@ -6,26 +6,32 @@
     <li class="nav-item">
       <a class="first-level-link" href="/index.html">Home</a>
     </li>
+    <li class="nav-item">
+      <a class="first-level-link" href="/components-patterns.html">Components and patterns</a>
+    </li>
     <li class="nav-item">
       <a
         class="first-level-link"
         href="{{(collections.all | allSamples)[0].url}}"
         >Samples</a
       >
     </li>
+    <li class="nav-item">
+      <a class="first-level-link" href="/top-task-1/">Top task 1</a>
```

Comment:
Needs to be a file link "xxx.html" to support desktop viewing.

Reply:
That was my comment, too. Same for task 2 overview link.

Reply:
this is done